format_spec/filters/float_scale.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Float Scaling Filter
+---
+
+The float scale filter stores its input data (with floating point type) as integer values for more compressed storage.
+For example, given `data` as a NumPy float64 array, scale = 2.13, offset = 15.445, and a specified byte width of 4:
+  ```
+  data = np.random.rand(npts)
+  data_b = data.view(np.float64)
+  new_data_b = np.zeros(npts, dtype=np.int32) # byte width is 4
+  scale = 2.13
+  offset = 15.445
+  
+  for i in range(0, len(data)):
+    new_data_b[i] = (data_b[i] - offset) / scale
```

Comment:
If you decide to leave this as is (see comment on an alternative example), make sure to add the rounding here.